src/websocket.c

```diff
@@ -389,7 +389,7 @@ static void send_pong(um_websocket_t *ws, const char* ping_data, int len) {
     buf.base = malloc(buf.len);
 
     buf.base[0] = WS_FIN | OpCode_Pong;
-    buf.base[1] = (char)(0x7f & len);
+    buf.base[1] = (char)(WS_MASK | (0x7f & len));
     if (ping_data != NULL && len > 0) {
         memcpy(buf.base + 2, ping_data, len);
```

Comment:
don't you need to add mask bytes and mask ping data?

Reply:
Yes I do. The one-liner above was my sanity check on wether MM server would stop terminating the connection on ping/pongs, so I can continue chasing another issue.  You also said you wanted to get some tweaks into a PR, so I published this for you to pile on.

Reply:
my contribution to the pile is comlete